tidb-lightning/tidb-lightning-configuration.md

```diff
@@ -402,11 +402,11 @@ sql-mode = "ONLY_FULL_GROUP_BY,NO_AUTO_CREATE_USER"
 max-allowed-packet = 67_108_864
 
 # Whether to use TLS for SQL connections. Valid values are:
-#  * ""            - force TLS (same as "cluster") if [tidb.security] section is populated, otherwise same as "false"
-#  * "false"       - disable TLS
+#  * ""            - force TLS (same as "cluster") if [tidb.security] section is populated, otherwise same as "false".
+#  * "false"       - same as "".
```

Comment:
```suggestion
#  - "": if the [tidb.security] section is populated, TLS is required (same as "cluster").
#  - "false": same behavior as "".
```

Reply:
- "": if the [tidb.security] section is populated, TLS is required (same as "cluster"). Otherwise, it will use an unencrypted connection.